jsx/src/App.jsx

```diff
@@ -23,34 +23,17 @@ const App = () => {
     <div className="resets">
       <Provider store={store}>
         <HashRouter>
-          <Switch>
+          <Routes>
```

Comment:
update of react-router-dom to v6 (needed for `useSearchParams`) necessitates changing `<Switch>` to `<Routes>` and a few other small changes. More info in the [upgrade docs](https://reactrouter.com/en/main/upgrading/v5).